hironx_ros_bridge/src/hironx_ros_bridge/hironx_client.py

```diff
@@ -835,6 +835,44 @@ def removeForceSensorOffset(self):
         self.rh_svc.removeForceSensorOffset()
 
     def getCurrentPose(self, lname=None, frame_name='WAIST'):
+        '''!@brief
+        Returns the current physical pose of the specified joint.
+        cf. getReferencePose that returns commanded value.
+
+        eg.
+        \verbatim
+             IN: robot.getCurrentPose('LARM_JOINT5')
+             OUT: [-0.0017702356144599085,
+              0.00019034630541264752,
+              -0.9999984150158207,
+              0.32556275164378523,
+              0.00012155879975329215,
+              0.9999999745367515,
+               0.0001901314142046251,
+               0.18236394191140365,
+               0.9999984257434246,
+               -0.00012122202968358842,
+               -0.001770258707652326,
+               0.07462472659364472,
+               0.0,
+               0.0,
+               0.0,
+               1.0]
+        \endverbatim
+
+        @type lname: str
+        @param lname: Name of the link.
+        @param frame_name str: set reference frame name (from 315.2.5)
+        @rtype: list of float
+        @return: Rotational matrix and the position of the given joint in
+                 1-dimensional list, that is:
+        \verbatim
+                 [a11, a12, a13, x,
+                  a21, a22, a23, y,
+                  a31, a32, a33, z,
+                   0,   0,   0,  1]
+        \endverbatim
+        '''
```

Comment:
version check -> https://github.com/fkanehiro/hrpsys-base/blob/master/python/hrpsys_config.py#L1156
hrpsys rises error where is hironx ignore the wrt frame, 
`if ":" in lname: -> https://github.com/DaikiMaekawa/rtmros_hironx/commit/1827e91a12441a8926666a57a082f658fa79c6c7, 
you can not set both ":" in lname and frame_name, need to add error check at -> https://github.com/fkanehiro/hrpsys-base/blob/master/python/hrpsys_config.py#L1154